pymc/gp/gp.py

```diff
@@ -531,8 +538,8 @@ def _build_conditional(
         rxx = y - mean_total(X)
         L = cholesky(stabilize(Kxx, jitter) + Knx)
         A = solve_lower(L, Kxs)
-        v = solve_lower(L, rxx)
-        mu = self.mean_func(Xnew) + pt.dot(pt.transpose(A), v)
+        v = solve_lower(L, rxx.T)
+        mu = self.mean_func(Xnew) + pt.dot(pt.transpose(A), v).T
```

Comment:
Extend `_build_conditional()` to multiple outputs when `marginal_likelihood()` returns `(n_outputs, X.shape[0])` size tensor